### ISSUE TITLE
Versioned releases

### ISSUE THREAD

**Issue by furlongm** [svartalf/unbound-telemetry#67]
Would it be possible to tag specific versions / cut releases, so that it's easier to track changes and create versioned artifacts for dpeloyment?

**Comment by svartalf** [comment]
Yes, it is generally what I want to reach - a proper release scheme, but at the moment it seems a bit unreasonable considering the "beta" status of application; with all used dependencies updated and frozen I'll be able to do that, but so far I'm blocked by a lack of PCKS8 support in the used TLS library (which is needed to read `unbound` certificates).

**Comment by furlongm** [comment]
Just wondering if there was any update on versioned releases? Even releases tagged as `alpha` or `beta` would be useful - and would also set expectations.
